test/test_manager.py

```diff
@@ -1198,3 +1198,21 @@ def assert_dd_appeared():
         assert manager_nospawn.c.eval("self.core.wmname") == (True, "LG3D")
     assert "dd" in manager_nospawn.c.groups()["S"]["windows"]  # First dropdown persists
     assert "dd" in manager_nospawn.c.groups()["1"]["windows"]  # Second orphans to group
+
+
+class CommandsConfig(Config):
+    screens = [
+        libqtile.config.Screen(
+            bottom=libqtile.bar.Bar([libqtile.widget.Systray()], 20),
+        )
+    ]
+
+
+@pytest.mark.parametrize("manager", [CommandsConfig], indirect=True)
+def test_windows_from_commands(manager):
+    manager.test_window("one")
+    assert len(manager.c.items("window")) == 2  # This command returns windows including bars
+    assert len(manager.c.windows()) == 1  # Whereas this one is just regular windows
+
+    # And the Systray is not present
+    assert "Systray" not in [c for w in manager.c.windows() for c in w["wm_class"]]
```

Comment:
Is the `Systray`'s `wm_class` ever set?

If I run `qtile cmd-obj -o widget systray -f info`, `wm_class` is `None` this assertion isn't really confirming that `Systray` is not in that list.

Reply:
Ah yeah, fixed